web/src/SidebarResources.tsx

```diff
@@ -94,6 +106,18 @@ function sortByHasAlerts(itemA: SidebarItem, itemB: SidebarItem): number {
   return Number(hasAlerts(itemB)) - Number(hasAlerts(itemA))
 }
 
+function matchesResourceName(item: SidebarItem, filter: string): boolean {
+  filter = filter.trim()
+  // this is functionally redundant but probably an important enough case to make its own thing
```

Comment:
🙃  is there a test case for this? (i.e. simulate change to "") should there be?

Reply:
ehh, there are a bunch of tests that have an empty filter, and I'm not sufficiently skeptical of material-ui's textfield to think it's worth going out of our way to test the state-change